lib/core/app/app_webview.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter_inappwebview/flutter_inappwebview.dart';
-import 'package:flutter_windows_webview/flutter_windows_webview.dart';
 import 'package:lantern/core/utils/common.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:webview_windows/webview_windows.dart';
```

Comment:
For Windows why can't we use flutter_inappwebview, which supports Windows already?

Reply:
I thought we had some issues with it on Windows, but I haven't tested it recently. Let me check if it works and if so happy to switch to it

Reply:
Ok we are using flutter_inappwebview on Windows now too